constr.mm

```diff
@@ -0,0 +1,139 @@
+$[ set.mm $]
+
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Constructible numbers
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  This section defines the set of constructible points as complex numbers which
+  can be drawn starting from two points (we take ` 0 ` and ` 1 ` ), and taking
+  intersections of circles and lines.
+
+  We initially define two sets:
+  <UL>
+  <LI>geometrically, by recursively adding the intersection points : this is
+    ` Constr ` .</LI>
+  <LI>algebraically, by using field extensions : this is ` ConstrAlg ` . </LI>
+  </UL>
+
+  We then proceed by showing that those two sets are equal.
+
+  This construction is useful for proving the impossibility of doubling the
+  cube ( ~ imp2cube ), and of angle trisection ( ~ imp3ang )
+$)
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Degree of a field extension
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    fldextdgirred.b $e |- B = ( Base ` F ) $.
+    fldextdgirred.0 $e |- .0. = ( 0g ` F ) $.
+    fldextdgirred.d $e |- D = ( deg1 ` F ) $.
+    fldextdgirred.o $e |- O = ( eval1 ` F ) $.
```

Comment:
```suggestion
    fldextdgirred.o $e |- O = ( F evalSub1 E ) $.
```
I think this implements @savask 's comment: the evaluation function now expects a polynomial with coefficients in E

Reply:
Yes, exactly!